src/fidasim.f90

```diff
@@ -9777,22 +10816,21 @@ subroutine pnpa_f
                     call attenuate(ri,rf,vi,states)
 
                     !! Store NPA Flux
-                    flux = (dtheta/(2*pi))*sum(states)*beam_grid%dv/nlaunch(i,j,k)
+                    flux = (dtheta/(2*pi))*sum(states)*pass_grid%r(i)*pass_grid%dv/nlaunch(i,j,k)
                     call store_npa(det,ri,rf,vi,flux,passive=.True.)
                 enddo gyro_range_loop
             enddo detector_loop
         enddo loop_over_fast_ions
     enddo loop_over_cells
     !$OMP END PARALLEL DO
 
-    npart = pnpa%npart
 #ifdef _MPI
-    call parallel_sum(npart)
+    call parallel_sum(pnpa%npart)
```

Comment:
The parallel sum for npart is already done in the `write_npa` subroutine. Doing it here also will break writing out the npa particles to file.